mne/realtime/base_client.py

```diff
@@ -3,13 +3,13 @@
 #
 # License: BSD (3-clause)
 
-import copy
+import threading
```

Comment:
Thanks @alexandrebarachant . I'd say let's stick to threading for now, get the API right, add an example, so people can pull and test. Then we can smooth out other details. Make it work, then make it nice :)